index.html

```diff
@@ -2828,46 +2828,43 @@ <h3><dfn>Writing content</dfn></h3>
         or writing <a>NDEF messages</a>.
       </p>
       <p>
-        To <dfn>validate external type</dfn> given |input:USVString|, run
+        To <dfn>split external type</dfn> given |input:USVString|, run
         these steps:
       </p>
-      <ol class=algorithm id="validate-external-type">
-        <li>
-          If |input:string| is not an [=ASCII string=], is empty, or its
-          length exceeds 255 bytes, return `false`.
-        </li>
+      <ol class=algorithm id="split-external-type">
```

Comment:
I don't know. To me it sounds more intuitive to have step 2 first since that is the real operation, and then step 1 as an error step of the former, otherwise someone might think they have to scan the string separately to check for `:`? 
No strong opinion here.

Reply:
People can implement it that way as the result is the same, this is about clear spec text :-)